app/assets/javascripts/views/history_view.js

```diff
@@ -5,41 +5,75 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addActivities', 'connectionError');
 
-    this.currentStory = {};
-
-    this.$el.html(this.template({id: 'history', name: I18n.t('projects.show.history')}));
+    this.$el.html(this.template({
+      id: 'history',
+      className:'historycolumn',
```

Comment:
yes,
`history_column` works like `epic_column`, `backlog_column`... Those are the `<td />`s containing the column header and the actual column.. 
`historycolumn` opposes to `storycolumn` which is the container holding the stories, and is is present on `ColumnView`s / Backlog, Chilly Bin, In Progress, Done